CHANGELOG.md

```diff
@@ -3,6 +3,10 @@
 All notable changes to this project will be documented in this file.
 
 --------------------------------------
+#### [1.0.31]
+
+[UPDATED] `appendControl` now allows controls to display when `alwaysOpen` is true
```

Comment:
Should we indicate the deps update as well?

Reply:
Added